packages/plugin-web-attribution-browser/src/helpers.ts

```diff
@@ -17,6 +17,17 @@ const domainWithoutSubdomain = (domain: string) => {
   return parts.slice(parts.length - 2, parts.length).join('.');
 };
 
+//Direct traffic mean no external referral, no UTMs, no click-ids, and no other customer identified marketing campaign url params.
+const isDirectTraffic = (current: Campaign) => {
+  const { referrer, referring_domain, ...currentCampaign } = current;
+  const currentCampaignWithValue = Object.keys(currentCampaign).filter(
+    (key) => currentCampaign[key] !== undefined && currentCampaign[key] !== '',
+  );
+  const noCampaign = Object.keys(currentCampaignWithValue).length === 0;
+
+  return !referrer && noCampaign;
```

Comment:
would it be as good as saying

```suggestion
  return Object.values(current).every(value => !value)
```

as in return true if every value in `current` is falsey otherwise return false?

Reply:
That's much clearer. I previously though the value could be boolean. Like this.